lib/real_savvy/jwt/abstract_token.rb

```diff
@@ -98,20 +98,24 @@ def to_share_token
         Base64.urlsafe_encode64(share_token_json, padding: false)
       end
 
+      def short_token
+        @token.split('.')[1]
+      end
+
       private
 
-      attr_reader :claims, :audience, :subject
+      attr_reader :audience, :subject
 
       def retrieve_claims
         raise NotImplementedError, "subclass did not define #retrieve_claims"
       end
 
       def retrieve_audience
-        @audience = ::RealSavvy::JWT::Config.retrieve_audience(claims) if claims
+        @audience = ::RealSavvy::JWT::Config.retrieve_audience(self) if claims && claims['aud']
```

Comment:
Cyclum gets this from the claims['iss'], but I can change the tests. 

I wonder if this will always hold true, like will tokens always have an `aud` claim
Will audience always of a consumer always use the `aud` claim?